app/dashboard/templates/shared/menu.html

```diff
@@ -24,13 +24,13 @@
 <div class="collapse navbar-collapse gc-navbar-collapse order-1 spacer-pb-6 spacer-pb-md-0" id="navbarSupportedContent">
 
   <nav class="navbar-nav m-auto gc-navbar-nav">
-    <div class="nav-item gc-nav-item dropdown px-md-2 px-lg-3 interior my-auto">
+    <!-- <div class="nav-item gc-nav-item dropdown px-md-2 px-lg-3 interior my-auto">
       <a class="nav-link gc-nav-link px-3 px-md-0" href="#" id="dropdownCommunity" role="button"
       data-menu="jtbd" aria-haspopup="true" aria-expanded="false">
         {% trans "What's Possible" %}
       </a>
       <div class="gc-mobile-spacer gc-mobile-spacer-jtbd gc-menu-background-clr d-none"></div>
-    </div>
+    </div> -->
```

Comment:
@giliomeejg wanna delete this code instead of leaving it in here ?

Reply:
Left it on purpose for reference but will remove.
PR to follow.